internal/syncer/connector/discord.go

```
@@ -0,0 +1,830 @@
+	req.Header.Set("Authorization", discordAuthorizationHeader(c.token))
+	req.Header.Set("Accept", "application/json")
+
+	for wait := 0; wait < discord429MaxWaits; wait++ {
+		resp, err := c.client.Do(req)
+		if err != nil {
+			return 0, err
+		}
+		if resp.StatusCode == http.StatusTooManyRequests {
+			retryAfter := discordRetryAfter(resp)
+			_, _ = io.Copy(io.Discard, io.LimitReader(resp.Body, 1024*1024))
+			resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)